torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -134,38 +159,55 @@ c10::IValue getFunctionTuple(const Function& func) {
                       {"types", Tup(types)},
                       {"register_size", register_size}});
 
-  return Tup({func.qualname().qualifiedName(), table});
+  if (!save_debug_info) {
+    return Tup({func.qualname().qualifiedName(), table});
+  } else {
+    // module debug info
+    std::vector<IValue> module_paths;
+    module_paths.reserve(op_module_paths.size());
+    for (auto& path : op_module_paths) {
+      module_paths.emplace_back(std::move(path));
+    }
+
+    auto debug_info = Table({{"module_debug_info", Tup(module_paths)}});
```

Comment:
Call this op_debug_info. The debug info contains module information but, debug info itself is for ops.